msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/DiscussionForumTool.java

```diff
@@ -9327,6 +9327,11 @@ public void setRankBeanList(List ranklist) {
 		this.rankBeanList.addAll(alist);
 	}
 
+	public boolean isRanksEnabled()
+	{
+		return ServerConfigurationService.getBoolean("msgcntr.forums.ranks.enable", true);
```

Comment:
It's fine to call this. this is the typical pattern. It's a very fast call that just adds a few lines to look up the value in a HashMap. The advantage of using the service is if changes dynamically (which is possible) then the updated value would get picked up. 

Maybe if you were running it a million times in a for loop you'd save it locally, but should be a small optimization.